modules/openapi-generator/src/main/resources/python-flask/requirements.mustache

```diff
@@ -1,10 +1,20 @@
 connexion >= 2.6.0; python_version>="3.6"
-connexion >= 2.3.0; python_version=="3.5"
-connexion >= 2.3.0; python_version=="3.4"
+# connexion 2.6 dropped support for python < 3.6
+connexion >= 2.3.0; python_version=="3.5" or python_version=="3.4"
```

Comment:
Please change this to `<= 2.3.0` I realize that I pinned this incorrectly when I fixed it.
The releases here: https://github.com/zalando/connexion/releases
Show that 2.3.0 is the last version that supports python 3.4 and 3.5

Reply:
Thank you for fixing this